src/applications/accreditation/21a/pages/06-background-information-chapter/conditionThatAffectsRepresentationDetails.js

```diff
@@ -0,0 +1,34 @@
+import {
+  textareaSchema,
+  textareaUI,
+  titleUI,
+} from '~/platform/forms-system/src/js/web-component-patterns';
+
+import ConditionThatAffectsRepresentationDetailsNote from '../../components/ConditionThatAffectsRepresentationDetailsNote';
+
+/** @type {PageSchema} */
+export default {
+  title: 'Condition that affects representation details',
+  path: 'condition-that-affects-representation-details',
+  depends: formData => formData.conditionThatAffectsRepresentation,
+  uiSchema: {
+    ...titleUI(
+      'Describe the condition or impairment and any treatment you receive now or in the past year',
+    ),
+    conditionThatAffectsRepresentationDetails: textareaUI(' '),
```

Comment:
Would it make sense to add a description here instead of just `(' ')`?

If so, could do the same in `conditionThatAffectsExaminationDetails: textareaUI(' '),`